tests/components/amberelectric/test_sensor.py

```diff
@@ -109,11 +109,11 @@ async def test_general_price_sensor(hass: HomeAssistant, setup_general: Mock) ->
     assert len(hass.states.async_all()) == 6
     price = hass.states.get("sensor.mock_title_general_price")
     assert price
-    assert price.state == "0.08"
+    assert price.state == "0.09"
     attributes = price.attributes
     assert attributes["duration"] == 30
     assert attributes["date"] == "2021-09-21"
-    assert attributes["per_kwh"] == 0.08
+    assert attributes["per_kwh"] == 0.09
     assert attributes["nem_date"] == "2021-09-21T08:30:00+10:00"
```

Comment:
Maybe it makes sense to do the general test changes in a separate PR

Reply:
Normally yes, but I had to change the generator to make the new code work, which in turn changed these results.